spec/requests/providers/addresses_request_spec.rb

```diff
@@ -71,7 +71,7 @@
 
       it 'renders the form again if validation fails' do
         post_request
-        expect(response.body).to include("Enter your client's address manually")
+        expect(response.body).to include(CGI.escapeHTML("Enter your client's address manually"))
```

Comment:
[Ditto](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/71/files#r232283856)